content/docs/user-guide/editor/asset-browser.md

```diff
@@ -0,0 +1,156 @@
+
+1. You can **duplicate** assets by doing one of the following in the **Asset Browser**:
+
+    - Use **Right Mouse Button** on an asset in any of the views to bring up the **context menu**, choose **Duplicate Asset**. 
+    - Select an asset in any of the views and use the **Ctrl + D** keyboard shortcut.
+
+1. You can **move** assets by doing one of the following in the **Asset Browser**:
+
+    - Use **Right Mouse Button** on an asset in any of the views to bring up the **context menu**, choose **Move To**. 
+    - Use the drag and drop functionality to drag an asset to the location of your choice.
+
+    ![Asset Browser Asset Management.](/images/user-guide/assetbrowser/asset-management.png)
+
+## Navigating Directories
+
+In the **Asset Browser**, you can navigate directories using the **Breadcrumbs Navigation Bar**, which indicates the currently selected directory.
+
+1. The **Breadcrumbs Navigation Bar** includes several useful features to make navigating directories easier, such as:
+
+- **Previous** and **Foward** navigation arrows that allow you to navigate through previously selected directories.
+- Clickable breadcrumbs to navigate through parent directories.
+     ![Asset Browser Click Navigate.](/images/user-guide/assetbrowser/breadcrumbs-click-navigate.png)
+- Clicking empty space in the **Breadcrumbs Navigation Bar** converts it to a **URL Bar** that includes the ability to edit, cut, copy, and paste file locations
+     ![Asset Browser Edit.](/images/user-guide/assetbrowser/breadcrumbs-edit.png)
+
+## Opening Assets
+
+1. Using the **Asset Browser**, you can open and edit assets by double clicking the **Left Mouse Button** on an asset.
+
+- To open your asset in the editor, drag and drop your asset from the **Asset Browser** into the **Entity Outliner** or **Viewport**.
+- To open your asset with an associated application, use **Right Mouse Button** on an asset to bring up the **context menu**, choose **Open with associated application...**
+- To view your asset in your operating system's **File Explorer**, use **Right Mouse Button** to bring up the **context menu** and choose **Open in Explorer**.
+
+## Search and Filter Assets
+
+For projects that have many assets, you can search and filter for the assets that you want. Enter text in the search filter box to find specific assets.
+
+**Search for assets and use type filters**
+
+1. In the **Asset Browser**, enter the name of the asset.
+
+1. To create a filter by **Asset Type**, click the filter icon.
+
+1. Enter the asset type in the search field that appears or scroll and select the asset type that you want. Any asset that has that specified asset type appears in the results.
+
+     The assets that appear are either **Material** assets or **Physics Material** assets.
+
+     ![Asset Browser Asset Type Filter.](/images/user-guide/assetbrowser/asset-type-filter.png)
+
+1. To clear search results, click **Clear**.
```

Comment:
- Made this into another section
- Switched to bullets instead of numbers, rearranged some of the points 

```suggestion
### Search field and type filters

To search and filter assets in the Asset Browser, do one of the following:
- Enter the name of the asset in the search field.

- Filter by **Asset Type** by clicking the filter icon and doing one of the following:

  - Enter the asset type in the filter's search field 
  - Scroll through the filter options and select the asset type that you want. 
  
  Any asset of the selected asset type appears in the results.

- Clear search results, by choosing **Clear**.

In the following example image, either **Material** or **Physics Material** assets appear.

![Asset Browser Asset Type Filter.](/images/user-guide/assetbrowser/asset-type-filter.png)
```